tests/indices/list/test_base.py

```diff
@@ -39,12 +46,23 @@ def documents() -> List[Document]:
     return [Document(doc_text)]
 
 
+@pytest.fixture
+def documents_with_overlap() -> List[Document]:
```

Comment:
the naming of this is a bit confusing, the overlap doesn't come from the document but rather the text splitter right? 

can we just define this as a `document` variable in the test itself